config/nutanix.yaml

```diff
@@ -0,0 +1,15 @@
+Cloud:
```

Comment:
Need also to add config/test_nutanix.yaml which should refer to nutanix.yaml, nutanix_flavors.yaml and nutanix_testcases.yaml.

Reply:
Thanks for the remind, added as described.